civicteam/civic-connect-android-public
# FIx documentation and cleanup dependency versions

# Work done

- Fix typo in the Documentation on the library name.
- Update all  gradle plugin and versions to work on latest stable Android Studio
- Change var to val in CivicConnectViewModel as it doesn't have to be a var
- Make tests more resilient
- Update build to save preprod and prod test apks

# How to test
* Observe two apks in circle artefacts
* Observe passing tests.

ref : WL=1259 Fixes issue #17
